nsdb-core/src/main/scala/io/radicalbit/nsdb/actors/MetricReaderActor.scala

```diff
@@ -319,6 +300,23 @@ class MetricReaderActor(val basePath: String, nodeName: String, val db: String,
           }
 
           shardResults.pipeTo(sender)
+
+        case Right(ParsedGlobalAggregatedQuery(_, _, _, limit, fields, _, _)) =>
```

Comment:
Looks very similar to the code in `ReadCoordinator`.
Might it be abstracted in a method?

Reply:
Agreed

Reply:
👍